### ISSUE TITLE
(suggestion) server console commands

### ISSUE THREAD

**Issue by KingTheGuy** [tomrhollis/vellum-for-EZ#2]
on my server we use a lot of /tag commands, would be nice if the other admins / users with administration or with a specific role can issue server commands from a text channel on discord.

**Comment by tomrhollis** [comment]
Interesting idea. So anyone with a specific role on discord would automatically have console access with this feature?

**Comment by KingTheGuy** [comment]
yup, would be nice to be able to use almost all of the commands, except for /kill ..that one can be really problematic, maybe with a bit more arguments like @a,@p,@s would be a no go, @e would require a radius just so none accidentally kills every mob in the server.

**Comment by tomrhollis** [comment]
That stuff could be done without restrictions by anyone with console access though. Seems like the line of defense there should probably be limiting access to the role on discord.

In any case, this idea is going on the list, though it will be a while before I get down to that point on the list! (Will post an outline of future plans soon)
